include/boost/geometry/util/series_expansion.hpp

```diff
@@ -0,0 +1,679 @@
+            c[3] = (n*((CT(2)-CT(2)*n)*n+CT(2))+CT(5))/CT(128);
+            c[4] = (n*(CT(3)*n+CT(11))+CT(12))/CT(512);
+            c[5] = (CT(10)*n+CT(21))/CT(1024);
+            c[6] = ((n-CT(3))*n+CT(2))/CT(32);
+            c[7] = (n*(n*(CT(2)*n-CT(3))-CT(2))+CT(3))/CT(64);
+            c[8] = (n*((-CT(6)*n-CT(9))*n+CT(2))+CT(6))/CT(256);
+            c[9] = ((CT(1)-CT(2)*n)*n+CT(5))/CT(256);
+            c[10] = (CT(69)*n+CT(108))/CT(8192);
+            c[11] = (n*((CT(5)-n)*n-CT(9))+CT(5))/CT(192);
+            c[12] = (n*(n*(CT(10)*n-CT(6))-CT(10))+CT(9))/CT(384);
+            c[13] = ((-CT(77)*n-CT(8))*n+CT(42))/CT(3072);
+            c[14] = (CT(12)-n)/CT(1024);
+            c[15] = (n*((CT(20)-CT(7)*n)*n-CT(28))+CT(14))/CT(1024);
+            c[16] = ((-CT(7)*n-CT(40))*n+CT(28))/CT(2048);
+            c[17] = (CT(72)-CT(43)*n)/CT(8192);
+            c[18] = (n*(CT(75)*n-CT(90))+CT(42))/CT(5120);
+            c[19] = (CT(9)-CT(15)*n)/CT(1024);
+            c[20] = (CT(44)-CT(99)*n)/CT(8192);
+            break;
+        case 8:
+            c[0] = (CT(1)-n)/CT(4);
+            c[1] = (CT(1)-n2)/CT(8);
+            c[2] = (n*((-CT(5)*n-CT(1))*n+CT(3))+CT(3))/CT(64);
+            c[3] = (n*((CT(2)-CT(2)*n)*n+CT(2))+CT(5))/CT(128);
+            c[4] = (n*(CT(3)*n+CT(11))+CT(12))/CT(512);
+            c[5] = (CT(10)*n+CT(21))/CT(1024);
+            c[6] = CT(243)/CT(16384);
+            c[7] = ((n-CT(3))*n+CT(2))/CT(32);
+            c[8] = (n*(n*(CT(2)*n-CT(3))-CT(2))+CT(3))/CT(64);
+            c[9] = (n*((-CT(6)*n-CT(9))*n+CT(2))+CT(6))/CT(256);
+            c[10] = ((CT(1)-CT(2)*n)*n+CT(5))/CT(256);
+            c[11] = (CT(69)*n+CT(108))/CT(8192);
+            c[12] = CT(187)/CT(16384);
+            c[13] = (n*((CT(5)-n)*n-CT(9))+CT(5))/CT(192);
+            c[14] = (n*(n*(CT(10)*n-CT(6))-CT(10))+CT(9))/CT(384);
+            c[15] = ((-CT(77)*n-CT(8))*n+CT(42))/CT(3072);
+            c[16] = (CT(12)-n)/CT(1024);
+            c[17] = CT(139)/CT(16384);
+            c[18] = (n*((CT(20)-CT(7)*n)*n-CT(28))+CT(14))/CT(1024);
+            c[19] = ((-CT(7)*n-CT(40))*n+CT(28))/CT(2048);
+            c[20] = (CT(72)-CT(43)*n)/CT(8192);
+            c[21] = CT(127)/CT(16384);
+            c[22] = (n*(CT(75)*n-CT(90))+CT(42))/CT(5120);
+            c[23] = (CT(9)-CT(15)*n)/CT(1024);
+            c[24] = CT(99)/CT(16384);
+            c[25] = (CT(44)-CT(99)*n)/CT(8192);
+            c[26] = CT(99)/CT(16384);
+            c[27] = CT(429)/CT(114688);
+            break;
+        }
+    }
+
+    /*
+    \brief Given the set of coefficients coeffs2[] evaluate on
+      C3 and return the set of coefficients coeffs1[].
+
+      Elements coeffs1[1] through coeffs1[SeriesOrder - 1] are set.
+    */
+    template <typename CT, size_t SeriesOrder>
+    static inline void evaluate_coeffs_C3(CT eps, CT coeffs1[], CT coeffs2[])
+    {
+        CT mult = 1;
+        int offset = 0;
+
+        // l is the index of C3[l].
```

Comment:
If this was separate type and the code was describing itself you wouldn't need to write this comment.